CHANGES.md

```diff
@@ -31,6 +31,80 @@ OpenSSL 3.6
 
 ### Changes between 3.5 and 3.6 [xx XXX xxxx]
 
+ * Added PCT for key import for SLH-DSA when in FIPS mode
+
+   *Dr Paul Dale*
+
+ * Added FIPS 140-3 PCT on DH key generation
+
+   *Nikola Pajkovsky*
+
+ * Added i2d_PKCS8PrivateKey(3) API to complement i2d_PrivateKey(3), the former
+   always outputs PKCS#8.
+
+   *Viktor Dukhovni*
+
+ * Implemented interleaved AES-CBC+HMAC-SHA algorithm on aarch64.
+
+   *Fangming Fang*
+
+ * Added NIST security categories for PKEY objects.
+
+   *Dr Paul Dale*
+
+ * Added notification when all stream FINs are acknowledged in QUIC. Introduced
+   `ossl_quic_channel_notify_flush_done()` so that once final FINs are ACKed,
+   the channel transitions to terminating and SSL_poll() signals completion.
+   This allows applications to progress shutdown reliably.
+
+   *Alexandr Nedvedicky*
+
+ * Fixed the synthesised `OPENSSL_VERSION_NUMBER`
```

Comment:
We seem to be putting `.` at the end of line for other entries, add it here and to the two initial entries as well.